openedx_tagging/core/tagging/import_export/parsers.py

```diff
@@ -201,24 +201,24 @@ class JSONParser(Parser):
     """
 
     format = ParserFormat.JSON
-    missing_field_error = FieldJSONError
-    empty_field_error = EmptyJSONField
+    missing_field_error: type[TagParserError] = FieldJSONError
+    empty_field_error: type[TagParserError] = EmptyJSONField
     inital_row = 0
 
     @classmethod
-    def _load_data(cls, file: BytesIO) -> Tuple[List[dict], List[TagParserError]]:
+    def _load_data(cls, file: BytesIO) -> tuple[list[dict], list[TagParserError]]:
         """
         Read a .json file and validates the root structure of the json
         """
         file.seek(0)
         try:
             tags_data = json.load(file)
         except json.JSONDecodeError as error:
-            return None, [
+            return [], [
```

Comment:
This was returning `(None, list)` in a clear violation of the declared return type `(list, list)`